### DISCUSSION TITLE
Java stuff

### DISCUSSION THREAD

**Discussion by tom-ricci** [XaverianTeamRobotics/CenterstageFTCrobotcontroller#52] [General]
@23dhicks Here's a video that's really good at teaching java (I've used it actually) so it should help you get the underlying concepts of java down which will make working with the sdk much easier (also this dude can explain things way better than me so I highly recommend watching the whole thing even though it's long)

https://youtu.be/eIrMbAQSU34

**Comment by 23dhicks** [comment]
Awesome thanks. I'll definitely watch it in a  bit.

On Wed, Nov 3, 2021 at 11:47 PM Thomas Ricci ***@***.***>
wrote:

> @23dhicks <https://github.com/23dhicks> Here's a video that's really good
> at teaching java (I've used it actually) so it should help you get the
> underlying concepts of java down which will make working with the sdk much
> easier (also this dude can explain things way better than me so I highly
> recommend watching the whole thing even though it's long)
>
> https://youtu.be/eIrMbAQSU34
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/XaverianTeamRobotics/FtcRobotController/discussions/52>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV3QFIINPAWIUGKGJFFXVMDUKIF6ZANCNFSM5HKP26JQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Donald Hicks
Class of 2023
Xaverian Brothers High School
800 Clapboardtree Street
Westwood, MA 02090
781-326-6392
***@***.***
www.xbhs.com
Instagram <http://instagram.com/xaverianhawks>| Facebook
<http://www.facebook.com/xhawks>| Twitter <http://www.twitter.com/xbhs>

-- 
CONFIDENTIALITY NOTICE
This transmission and the attached hereto may 
contain confidential information belonging to Xaverian Brothers High School 
which is intended solely for the named recipient. If you are not the named 
recipient, you are hereby notified that any unauthorized use, disclosure, 
duplication and/or distribution of the following contents are strictly 
prohibited. If you have received this transmission in error, please notify 
us immediately.
